manifests/node/kubelet.pp

```diff
@@ -137,10 +137,7 @@
 
   file { '/etc/modules-load.d/k8s':
     ensure  => $ensure,
```

Comment:
for future reference: we should update this with our kmod module.

Reply:
#8 is there to keep track